pkg/cluster/delete.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/azureclient"
 	"github.com/Azure/ARO-RP/pkg/util/azureerrors"
 	"github.com/Azure/ARO-RP/pkg/util/dns"
+	utilnet "github.com/Azure/ARO-RP/pkg/util/net"
```

Comment:
minor preference: naming to follow stringutils, so netutils instead of utilnet

Reply:
Good point. I have chosen utilnet as we already use it in a few places:
- ./pkg/gateway/gateway_test.go
- ./pkg/gateway/https.go
- ./pkg/proxy/proxy.go

Maybe this should change in the future, not sure. 🙂